lib/vm/src/memory.rs

```diff
@@ -431,6 +436,16 @@ impl LinearMemory for VMSharedMemory {
         let forked = Self::duplicate(self)?;
         Ok(Box::new(forked))
     }
+
+    // Add current thread to waiter list
+    fn do_wait(&mut self, dst: u32, timeout: i64) -> u32 {
```

Comment:
This is relatively minor, but can we keep using `NotifyLocation` here for better code clarity? Since we already have the type.

Would make the code more readable.

If you change it... since we convert the timeout to a `Duration` later anyway , I'd also take a `Duration` here so conversion happens at the earliest point.

Reply:
Ok, I'll do that

Reply:
done